worlds/cvcotm/regions.py

```diff
@@ -0,0 +1,185 @@
+from .data import lname
+from typing import List, Optional
+
+# # #    KEY    # # #
+# "locations" = A list of the Locations to add to that Region when adding said Region.
+# "entrances" = A dict of the connecting Regions to the Entrances' names to add to that Region when adding said Region.
+cvcotm_region_info = {
```

Comment:
You could use a TypedDict here
```suggestion
class RegionInfo(TypedDict, total=False):
    locations: List[str]
    entrances: Dict[str, str]


cvcotm_region_info: Dict[str, RegionInfo] = {
```